Packages/vcs/Lib/VTKPlots.py

```diff
@@ -357,7 +358,7 @@ def createRenderer(self, *args, **kargs):
         return ren
 
     def update(self, *args, **kargs):
-        self._lastSize = -1
+        self._lastSize = None
```

Comment:
@chaosphere2112 I added a missing png for a test you added. Can you make sure it is the correct image? Thanks.